shared/actions/fs/platform-specific.desktop.js

```diff
@@ -391,7 +391,10 @@ function* platformSpecificSaga(): Saga.SagaGenerator<any, any> {
     )
   }
   yield* Saga.chainAction<FsGen.OpenAndUploadPayload>(FsGen.openAndUpload, openAndUpload)
-  yield* Saga.chainAction<FsGen.UserFileEditsLoadPayload>(FsGen.userFileEditsLoad, loadUserFileEdits)
+  yield* Saga.chainAction<FsGen.UserFileEditsLoadPayload | FsGen.KbfsDaemonRpcStatusChangedPayload>(
+    [FsGen.userFileEditsLoad, FsGen.kbfsDaemonRpcStatusChanged],
+    loadUserFileEdits
```

Comment:
Sorry for not mentioning this earlier, but this would trigger it even on disconnected. Might wanna do something like `state.fs.kbfsDaemonStatus.rpcStatus === 'connected' &&` [here](https://github.com/keybase/client/blob/64eca01005d033f2b90ab63143fb2811c8fcf971/shared/actions/fs/platform-specific.desktop.js#L358)